src/hooks/queries/useFetchCartItems/useFetchCartItems.ts

```diff
@@ -28,6 +29,10 @@ const useFetchCartItems = () => {
     if (error) showToast(error.message);
   }, [error]);
 
+  useEffect(() => {
+    if (isPaused && !navigator.onLine) showToast(ERROR_MESSAGES.OFFLINE);
```

Comment:
`window.navigator.onLine` 사용 시 다음의 내용도 참고해주시면 좋을 것 같아요 :)

> Browsers implement this property differently.

> In Chrome and Safari, (...중략...) You could be getting false positives, such as in cases where the computer is running a virtualization software that has virtual ethernet adapters that are always "connected." Therefore, if you really want to determine the online status of the browser, you should develop additional means for checking.

- 참고 링크
  - https://developer.mozilla.org/en-US/docs/Web/API/Navigator/onLine

> In previous versions, navigator.onLine was used to determine the network status. However, it doesn't work well in Chromium based browsers. There are [a lot of issues](https://bugs.chromium.org/p/chromium/issues/list?q=navigator.online) around false negatives, which lead to Queries being wrongfully marked as offline.

> To circumvent this, we now always start with online: true and only listen to online and offline events to update the status.

> This should reduce the likelihood of false negatives, however, it might mean false positives for offline apps that load via serviceWorkers, which can work even without an internet connection.

  - https://tanstack.com/query/latest/docs/reference/onlineManager